pub/get.php

```diff
@@ -19,7 +19,7 @@
 
 $isAllowed = function ($resource, array $allowedResources) {
     foreach ($allowedResources as $allowedResource) {
-        if (0 === stripos($resource, $allowedResource)) {
+        if (0 === stripos($resource, 'media/' . $allowedResource)) {
```

Comment:
Implementation will be broken if the resource will be configured out of `media` folder.